backend/controllers/prisonerProfile/prisonerLocationHistory.js

```diff
@@ -72,7 +79,7 @@ module.exports = ({ prisonApi, whereaboutsApi, caseNotesApi, logError }) => asyn
       caseNotesApi,
       whereaboutsApi
     )
-    const whatHappenedDetailsToShow = isBlank(whatHappenedDetails) ? 'Not entered' : whatHappenedDetails
+    const whatHappenedTrimmedToSize = trimString(whatHappenedDetails, whatHappenedMaxLength)
```

Comment:
Why are you trimming the comment?

Reply:
due to comment in the ticket, but now removed